sqlserver/datadog_checks/sqlserver/utils.py

```diff
@@ -25,6 +25,19 @@ def construct_use_statement(database):
     return 'use [{}]'.format(database)
 
 
+def is_statement_proc(text):
+    if text:
+        t = text.upper().split()
```

Comment:
taking the split based on https://github.com/DataDog/integrations-core/pull/12798#discussion_r956113198

Reply:
re: top 500 chars https://github.com/DataDog/integrations-core/pull/12798#discussion_r956128223

I think it is much more probable that we would get false positives in SQL tens of thousands of characters long than miss some procedures which have long comments.

Reply:
okay, ill take the first 500 chars 🤷